tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1111,6 +1111,74 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x1c - Get Attribute
+
+		Controller Index:	<controller id>
+		Command parameters:	Start Handle (2 octets)
+					End Handle (2 octets)
+					Attribute Type (1 octet)
+					UUID_Length (1 octet)
+					UUID (2 or 16 octets)
+		Response parameters:	Attribute Type (1 octet)
+					Object (variable)
+
+		Valid Attribute Type parameter values:
+					0x01 = Primary Service
+					0x02 = Secondary Service
+					0x03 = Included Service
+					0x04 = Characteristic
+					0x05 = Characteristic Value
+					0x06 = Descriptor
+
+		Valid UUID_Length parameter values:
+					0x02 = UUID16
+					0x10 = UUID128
+
+		Object Service is defined as:
+					Service Type (1 octet)
+					Start_Group_Handle (2 octets)
+					End_Group_Handle (2 octets)
+					UUID_Length (1 octet)
+					UUID (2 or 16 octets)
+
+		Valid Service Type parameter values:
+					0x01 = Primary Service
+					0x02 = Secondary Service
+
+		Object Included Service is defined as:
+					Handle (2 octets)
+					Object Service
+
+		Object Characteristic is defined as:
+					Handle (2 octets)
+					Value_Handle (2 octets)
+					Properties (1 octet)
+					UUID_Length (1 octet)
+					UUID (2 or 16 octets)
+
+		Object Characteristic Value is defined as:
+					Handle (2 octets)
+					Permissions (1 octet)
+					Value_Len (2 octets)
```

Comment:
I don't think the database necessarily have its value stored locally, for BlueZ this might incur in D-Bus traffic if the attribute happens to be in the application side so I wonder if this should really be supported. IMO just the attribute definitions should be fine, if PTS requires to check the values then perhaps a separate command should be added.

Reply:
OK. Makes sense. I would merge Characteristic and Characteristic Value too. So I'll end up with additional Permissions value.